draft-ietf-quic-multipath.md

```diff
@@ -1452,21 +1474,29 @@ MAX_PATHS Frame {
 
 MAX_PATHS frames contain the following field:
 
-Maximum Path Identifier:
+Maximum Paths:
 : A count of the cumulative number of path that can be opened
-over the lifetime of the connection. This value cannot exceed 2^32-1, as it is not
-possible to encode Path IDs larger than 2^32-1. Receipt of a frame that permits
-opening of a path with Path Identifier larger than this limit MUST be treated
+over the lifetime of the connection.
+
+The endpoint receiving this frame MAY send MP_NEW_CONNECTION_ID frames
+with Path IDs strictly lower than twice the Maximum Paths.
+For example, if an endpoint received a Maximum Paths value of 2,
+it may send MP_NEW_CONNECTION_ID with Path ID 0 and 2, but not higher.
```

Comment:
A small suggestion to make this clearer (and avoid the potential reader ambiguity of the "2" value)
```suggestion
For example, if an endpoint received a Maximum Paths value of 3,
it may send MP_NEW_CONNECTION_ID with Path ID 0, 2 and 4, but not higher.
```

Reply:
Yes, good idea.